eng/pipelines/templates/jobs/azuresdkpartnerdrops-to-nugetfeed.yml

```diff
@@ -44,6 +44,10 @@ parameters:
     type: boolean
     default: true
 
+variables:
+  - name: skipComponentGovernanceDetection
```

Comment:
Would it be better to import [globals.yml](https://github.com/Azure/azure-sdk-tools/blob/main/eng/pipelines/templates/variables/globals.yml) here or will that add too many other conflicting variable overrides?

Reply:
It should be fine to import globals.yml